### PR TITLE
First pass homepage (behind feature flag)

### PR DESCRIPTION
**Pull request by johnnyd-eth**
Apologies for the size of this PR. Had a lot of time without WIFI so was banging out as much HTML as I could get done without testing, which meant covering a lot of the homepage at once.

**Summary:**
- Added a new `Landing` component hidden behind feature flag `'newLandingPage'`
- Built all sections of the design (https://www.figma.com/file/UE4g6PFX3d7odydhzAeD5j/JBM-Website-Updates-March-2023?node-id=503-153921&t=0OG7zJjPjN2BvZzs-0) except for 'Juicy picks'
- Images are resized and imageOptimed

**Known issues:** (which I will tackle in follow-ups)
- Icons should be SVGs
- I haven't gone through the UI with a fine-tooth-comb yet so expecting bugs there. Don't worry about them I'd say, I will pick them up with Strath in further iterations. **Probably just code, not UI review is all that's needed here**

### CODE REVIEW

File: public/assets/trust-icon.png
Comment (tomquirk) [inline public/assets/trust-icon.png:1]: Svgs for these?

File: src/pages/home/BuiltForSection/BuiltForCard.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  heading,
+  subheading,
+}: {
+  imageSrc: string
+  imageAlt: string
+  heading: string | JSX.Element
+  subheading: string | JSX.Element
+}) {
+  return (
+    <Col md={6} xs={24}>
```
Comment (tomquirk) [inline src/pages/home/BuiltForSection/BuiltForCard.tsx:16]: Is there a tailwind -only solution here?

File: src/pages/home/BuiltForSection/BuiltForSection.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      <SectionHeading
+        heading={<Trans>Built for ideas like yours</Trans>}
+        subheading={
+          <Trans>
+            Juicebox is as versatile as you are, build anything from an NFT
+            project to a boutique crypto lawfirm and everything in between.
+          </Trans>
+        }
+      />
+      <Row gutter={DEFAULT_HOMEPAGE_GUTTER}>
```
Comment (tomquirk) [inline src/pages/home/BuiltForSection/BuiltForSection.tsx:20]: Is there a tailwind solution here?

File: public/assets/trust-icon.png
Comment (johnnyd-eth) [reply public/assets/trust-icon.png:1]: Yep for sure. Was planning in next iter

File: src/pages/home/BuiltForSection/BuiltForSection.tsx (hunk 2 above)
Comment (johnnyd-eth) [reply src/pages/home/BuiltForSection/BuiltForSection.tsx:20]: Are we trying to ditch `Row` and `Col` altogether?

**Comment by johnnyd-eth** [comment]
oG6-jMxHBHVAe3uNFRFjmE-guWcU6PC@BrRXzxNjzvyZsqpPcjGKR-tBN6xn

**Pull request merged**